api/src/main/java/org/openmrs/api/UserService.java

```diff
@@ -110,12 +111,12 @@
 	public User getUserByEmail(String email);
 	
 	/**
-	 * Gets user using user activation key
-	 * @param activationKey User's activation(token) for password reset 
-	 * @return requested user
+	 * Gets User using user activation key(hash of token)
+	 * @param token User's (token) for password reset 
+	 * @return requested User with associated hashed of token as activation key
```

Comment:
This method on the service layer should not mention anything to do with hashing, the caller provides the raw token and should only mention token

Reply:
It's also okay to call it an activation key